tools/template/README.md

```diff
@@ -0,0 +1,43 @@
+# wtstats template
+
+This subfolder contains the code to generate the `wtstats.html.template` file needed by the Python script `wtstats.py`, which is located in the `./tools` directory. 
+
+The template is generated as a node.js "single page" application using several open source frameworks, see Dependencies below. The `wtstats.py` Python script parses stats files, transforms the data and inserts it as JSON into the template. The resulting HTML page can be openend in the browser.
+
+### Build Process
+
+To build the template, follow these steps: 
+
+1. change into the `./tools/template` directory
+2. run `npm install` to install all missing dependencies. 
```

Comment:
I wonder if some of this information how to install it should be in the documentation, rather than a README?  Maybe with a screen shot to show what it's capable of.